data/SarthakNegi.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Sarthak Negi",
+  "type": "personal",
+  "bio": "devOps enthusiast , learning webd , open source advocate!",
+  "tags": ["Sarthak", "Open Source", "GitHub", "Community", "Collaboration"],
+  "links": [
+    {
+      "name": "Follow me in twitter and linkedin",
+      "url": "https://twitter.com/sarthaknegi009",
+      "icon": "FaGlobe"
+    },
+    {
+      "name": "Contribute to Open Source in our GitHub Organisation",
+      "url": "https://github.com/sarthaksarthak9",
+      "icon": "FaGithub"
+    },
+
+    {
+      "name": "LinkedIn Organisation",
+      "url": "https://www.linkedin.com/in/sarthak-negi-4923a6259/",
+      "icon": "FaLinkedin"
+    },
```

Comment:
Please remove the extra **'Comma'** from this line :+1: 
```Suggestion
    }
```

Reply:
ok , thank you for reviewing !!